Dockerfile

```diff
@@ -1,56 +1,129 @@
-FROM trailofbits/polytracker-llvm:bbdb72d136af011db9710a4d44a79e6d5059cc1a
+FROM ubuntu:jammy as base
 
 LABEL org.opencontainers.image.authors="evan.sultanik@trailofbits.com"
-ENV DEBIAN_FRONTEND=noninteractive
-RUN apt-get -y update  && \
-      apt-get install -y \
-            ninja-build                                     \
-            python3-pip                                     \
-            python3.8-dev                                   \
-            libgraphviz-dev                                 \
-            libjpeg-dev                                     \
-            graphviz                                        \
-            vim                                             \
-            file                                            \
-            gdb                                             \
-            libncurses5-dev                                 \
-            apt-transport-https                             \
-            ca-certificates                                 \
-            libstdc++-10-dev
+
+ARG BUILD_TYPE="Release"
+ARG PARALLEL_LINK_JOBS=1
+ARG LLVM_TARGET_NAME=X86
+
+RUN DEBIAN_FRONTEND=noninteractive apt-get -y update
+RUN DEBIAN_FRONTEND=noninteractive apt-get -y install \
+  ninja-build                                         \
+  python3-pip                                         \
+  python3.8-dev                                       \
+  python-is-python3                                   \
+  ca-certificates                                     \
+  libstdc++-10-dev                                    \
+  golang                                              \
+  clang-12                                            \
+  cmake                                               \
+  git                                                 \
+  file
+
+RUN update-alternatives --install /usr/bin/opt opt /usr/bin/opt-12 10
+RUN update-alternatives --install /usr/bin/llvm-link llvm-link /usr/bin/llvm-link-12 10
+RUN update-alternatives --install /usr/bin/llvm-ar llvm-ar /usr/bin/llvm-ar-12 10
+RUN update-alternatives --install /usr/bin/clang clang /usr/bin/clang-12 10
+RUN update-alternatives --install /usr/bin/clang++ clang++ /usr/bin/clang++-12 10
+
+WORKDIR /
+RUN git clone --depth 1 --branch llvmorg-13.0.0 https://github.com/llvm/llvm-project.git
+RUN git clone --depth 1 --branch master https://github.com/trailofbits/blight.git
+
+RUN pip3 install pytest /blight
 
 RUN update-ca-certificates
 
-RUN apt-get install -y software-properties-common && \
-      # deadsnakes PPA allows us to install python3.10 on focal
-      add-apt-repository ppa:deadsnakes/ppa && \
```

Comment:
thanks for removing this, I was hoping to get around to it once we were fully away from focal, so this is perfect